frameworks/helloworld/tests/test_sanity.py

```diff
@@ -48,6 +49,24 @@ def test_install():
     check_running(FOLDERED_SERVICE_NAME)
 
 
+# Note: presently the mesos v1 api does _not_ work in strict mode.
+# As such, we expect this test to fail until it does in fact work in strict mode.
+@pytest.mark.sanity
+@pytest.mark.skipif(os.environ.get("SECURITY") != "strict")
```

Comment:
I'd originally meant "We should run this in strict and see that it fails"

But, we should actually run this in all modes.